cmd/gitbase/server.go

```diff
@@ -44,12 +45,12 @@ func (c *cmdServer) buildDatabase() error {
 		logrus.WithField("dir", c.Git).Debug("added folder containing git repositories")
 	}
 
-	var err error
+	c.pool = gitbase.NewRepositoryPool()
```

Comment:
so, now you can have mixed repositories on the same execution, standard ones and siva ones?

Reply:
Yup. I can restrict that if we want to only have one type

Reply:
it's ok, LGTM!